package.json

```diff
@@ -9,6 +9,9 @@
     "contracts"
   ],
   "scripts": {
+    "test:setup": "docker compose up --detach && pushd contracts && forge compile && popd",
+    "test:e2e": "pushd test && bun test:e2e && popd",
+    "test": "bun test:setup && bun test:e2e",
```

Comment:
So now this runs full setup and test no makefile

Reply:
Nice